src/main/main.cpp

```diff
@@ -186,6 +186,18 @@ checkXDRFileIdentity()
         {
             continue;
         }
+        // Temporarily (until we have cut the next minor release) disable the
+        // check for the XDR changes necessary for the minor release. These
+        // are mostly binary compatible with the Rust XDR, and the incompatible
+        // part (ScAddress change) is actually supposed to be not decodable on
+        // the Rust side.
+        if (cpp.first.filename() == "Stellar-types.x" ||
+            cpp.first.filename() == "Stellar-contract.x" ||
+            cpp.first.filename() == "Stellar-ledger-entries.x" ||
+            cpp.first.filename() == "Stellar-ledger.x")
+        {
+            continue;
+        }
```

Comment:
instead of skipping, would it be better to statically assert the hashes of these files, on both sides?